accessibility-guidelines.md

```diff
@@ -0,0 +1,87 @@
+
+#### A whole different UX
+
+The markup might be correct and all the correct ARIA tags can be utilized. But if for example an application relies on color to convey important information, that will be lost on someone who is color blind, unless it is supplemented in other ways. This is not something that can be caught by an automated test but requires context and the insight of a human.
+
+Likewise consider someone using a screen reader to search through a list containing hundreds of records based on date. If the date text reads "Created on March 25", they will have to wait for "created on" to be read aloud for every record. If the structure was instead altered to be "March 25, created on" the important information would be read aloud first.
+
+In many ways working with accessibility is improving a whole different UX than the one people without disabilities is used to. It has its own patterns, pains and ways of being used.
+
+#### More than screen readers
+
+Many developers will be aware of the fact that their application might be accessed by users depending on screen readers. If this is the only focus in terms of accessibility, the application is mainly made more accessible for those with disabilities affecting sight.
+
+However there is a lot of disabilities to consider when improving accessibility. Someone who has trouble hearing or is deaf will need captions to understand videos. A person with motor problems will have a hard time clicking an element within a timespan.
+
+Someone with [vestibular or seizure](https://webaim.org/articles/seizure/) disorder would prefer that their [preference for reduced motion](https://web.dev/prefers-reduced-motion/) is respected. If not they might get nauseous or have an outright seizure if excessive motion such as parallax scrolling or strobelight like effects is used.
+
+#### An ego-centric argument for accessibility
+
+To conclude accessibility is many things. Various aspects of your application ranging from the markup to design is affected by it.
+
+And it might seem overwhelming... So besides helping those with disabilities to access and use the digital tools we all rely on in the 21st century; why care as a developer without any disabilities?
+
+So here is a more ego-centric argument for caring about accessibility: you will most likely be disabled to some degree in the future. Apologies if it sounds bleak.
+
+You might break your dominant arm causing you to rely on keyboard navigation. More severely you could get into a car accident paralyzing you from the neck down. Moreover your sight, motorskills and reaction time will most likely decrease as you get older.
+
+In any case would you be able to rely on the digital tools you do now in a world where developers does not consider accessibility?
+
+## Kirby's role
```

Comment:
Consider a comparison of the role of Kirby components to HTML elements; they can be used both in a few "correct" ways and also in a lot of completely inaccessible ways

Reply:
I've attempted to add it here: https://github.com/kirbydesign/designsystem/pull/1851/commits/682a4a07ee5719096762df9f4b4fa7fcabbe7241